src/Symfony/Component/AssetMapper/ImportMap/PackageUpdateInfo.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\ImportMap;
+
+class PackageUpdateInfo
+{
+    public function __construct(
+        public readonly string $importName,
+        public readonly string $url,
+        public readonly string $currentVersion,
+        public ?string $majorUpdate = null,
+        public ?string $minorUpdate = null,
+        public ?string $patchUpdate = null,
+    ) {
+    }
+
+    public function addMajorUpdate(string $version): void
+    {
+        if (null === $this->majorUpdate) {
+            $this->majorUpdate = $version;
+        }
+    }
+
+    public function addMinorUpdate(string $version): void
+    {
+        if (null === $this->minorUpdate) {
+            $this->minorUpdate = $version;
+        }
+    }
```

Comment:
A package will only have a major OR minor OR patch update, but never more than one, right? Perhaps we just have one property - e.g. `latestVersion` with some internal "type" flag to track if it's a major, minor, patch.